sbin/submitty_autograding_shipper.py

```diff
@@ -407,67 +453,76 @@ def grade_queue_file(my_name, which_machine,which_untrusted,queue_file):
     :param which_untrusted: specific untrusted user for this autograding job
     """
 
-    my_dir,my_file=os.path.split(queue_file)
-    pid = os.getpid()
+    my_dir, my_file = os.path.split(queue_file)
     directory = os.path.dirname(os.path.realpath(queue_file))
     name = os.path.basename(os.path.realpath(queue_file))
     grading_file = os.path.join(directory, "GRADING_" + name)
 
     # Try to short-circuit this job. If it's possible, then great! Clean
     # everything up and return.
     if try_short_circuit(queue_file):
-        grading_cleanup(queue_file, grading_file)
+        grading_cleanup(my_name, queue_file, grading_file)
```

Comment:
What's going on here with the new parameter?

Reply:
(Can't really link to the direct line since Github collapses the diff of the file anyway, so bear with me)

In the `grading_cleanup` function, there are two exception handlers; one handling errors on removing queue files, and the other handling errors on removing grading files. These print a message both to stdout and the log, which reference a `my_name` variable that didn't exist in the function -- presumably due to the parameter never actually being added to the function. I added a `my_name` param to the function signature to rectify this, which is why there's a few `grading_cleanup` diffs in this style.